libgiza/config.py

```diff
@@ -140,9 +140,9 @@ def get_dict_value(v):
 
         for key, value in self.state.items():
             if safe in (True, None):
-                if key.startswith('_'):
+                if key != "_id" and key.startswith('_'):
                     continue
-                elif key in ('pass', 'password', 'token', 'key', 'secret'):
+                elif 'pass' in key or key in ('token', 'key', 'secret'):
```

Comment:
I want to also catch things like "email_password" or similar. But we miss "passengers" and other things. 

I'll try and find something thats more specific.